### PR TITLE
Use staticcheck

### PR DESCRIPTION
**Pull request by ysksuzuki**
This PR modifies the followings
- Use staticcheck instead of golint
- Flatten mtest directory structure to avoid staticcheck warning.
- Fix other staticcheck errors
- Return error when a user passes an invalid sabactl subcommand.

### CODE REVIEW

File: e2e/sabactl_test.go [hunk 1]
```diff
@@ -19,21 +19,15 @@ import (
 
 const (
 	// ExitSuccess represents no error.
-	ExitSuccess subcommands.ExitStatus = subcommands.ExitSuccess
```
Comment (ymmt2005) [inline e2e/sabactl_test.go:22]: Please get rid of `github.com/google/subcommands` as it was replaced with cobra.

File: models/etcd/dhcp.go [hunk 2]
```diff
@@ -67,6 +67,9 @@ type leaseUsage struct {
 }
 
 func (l *leaseUsage) MarshalJSON() ([]byte, error) {
+	if l.hwMap == nil {
+		return json.Marshal(make(map[string]leaseInfo))
+	}
```
Comment (ymmt2005) [inline models/etcd/dhcp.go:72]: unnecessary.

File: mtest/Makefile [hunk 3]
```diff
@@ -17,14 +17,8 @@ COREOS_INITRD := $(abspath .)/flatcar_production_pxe_image.cpio.gz
...
-SUITE = ./functions
-ifeq ($(findstring /,$(SUITE)),)
-  SUITE_PACKAGE = ./$(SUITE)
-else
-  SUITE_PACKAGE = $(SUITE)
-endif
-
-export BRIDGE_ADDRESS HOST1 HOST2 HOST3 WORKER1 WORKER2 PLACEMAT GINKGO SUITE_PACKAGE
+
+export BRIDGE_ADDRESS HOST1 HOST2 HOST3 WORKER1 WORKER2 PLACEMAT GINKGO
```
Comment (ymmt2005) [inline mtest/Makefile:21]:
```suggestion
export HOST1 HOST2 HOST3 WORKER1 WORKER2 PLACEMAT GINKGO
```

**Review by ymmt2005 — CHANGES_REQUESTED** [review]

File: models/etcd/dhcp.go [hunk 4]
```diff
@@ -76,6 +76,10 @@ func (l *leaseUsage) UnmarshalJSON(data []byte) error {
 		return err
 	}
 
+	if l.hwMap == nil {
+		l.hwMap = make(map[string]leaseInfo)
+	}
+
```
Comment (ysksuzuki) [inline models/etcd/dhcp.go:82]: In some cases, hwmap will be nil, so we initialize it here.

Comment (ymmt2005) [reply models/etcd/dhcp.go:82]: Why do you want to initialize it _here_?
I think it can be initialized where a new key is stored in this map.

Comment (masa213f) [reply models/etcd/dhcp.go:82]: I think it is better to allocate the map when initializing the "leaseUsage" structure.
In this case, in effect,  this unmarshaller initialize the structure, we're allocating the map here.

**Review by ymmt2005 — APPROVED** [review]
LGTM

**Pull request merged**